Robot/InfiniteRecharge/src/main/java/frc/robot/commands/SpinShooterMotorsCommand.java

```diff
@@ -24,7 +24,7 @@ public SpinShooterMotorsCommand() {
   public void initialize() {
     logger.info("got to Shooter motor Activate");
     // TODO: Retrieve the desired motor speed
-    RobotContainer.getInstance().shooter.SpinMotor(10000);
+    RobotContainer.getInstance().shooter.SpinMotor(RobotContainer.getInstance().Dashboard.getShooterSetSpeed());
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.